recipes/geotrans/all/CMakeLists.txt

```
@@ -0,0 +1,60 @@
+cmake_minimum_required(VERSION 3.4)
```

```suggestion
cmake_minimum_required(VERSION 3.8)
```

Some features in your custom CMakeLists require CMake 3.8+